src/client/sound/al_extensions.h

```diff
@@ -0,0 +1,38 @@
+/*
+Minetest
+Copyright (C) 2023 DS
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this program; ifnot, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#pragma once
+
+#include "al_helpers.h"
+
+namespace sound {
+
+/**
+ * Struct for AL and ALC extensions
+ */
+struct ALExtensions
+{
+	explicit ALExtensions(const ALCdevice *deviceHandle [[maybe_unused]]);
+
+#ifdef AL_SOFT_direct_channels_remix
```

Comment:
Unnecessary optimization. Does not check for correct syntax on all builds.

Reply:
Same as above.